src/device/router_device_routing.erl

```diff
@@ -863,6 +864,7 @@ lookup_bf(Key) ->
     Pid :: pid()
 ) -> ok | {error, any()}.
 packet(
+    _Device,
```

Comment:
This clause is for joins, and we request the device from Console by EUI to make sure we have the most up to date version, we want to continue to do that. So we ignore the passed device here.